robottelo/cli/repository.py

```diff
@@ -13,8 +13,10 @@
 
     create                        Create a repository
     delete                        Destroy a repository
+    export                        Export a repository
     info                          Show a repository
     list                          List of repositories
+    remove-content
```

Comment:
Sure, good catch. This is what "hammer repository -h" gave me, so I should report this to hammer devs... Anyways, I'll update it here.